easy_profiler_core/CMakeLists.txt

```diff
@@ -159,7 +159,12 @@ set(SOURCES
     ${INCLUDE_FILES}
 )
 
-add_library(easy_profiler ${SOURCES} resources.rc)
+set(LIBRARY_TYPE SHARED)
+if (NOT BUILD_SHARED_LIBS)
+    set(LIBRARY_TYPE STATIC)
+endif ()
+
+add_library(easy_profiler ${LIBRARY_TYPE} ${SOURCES} resources.rc)
```

Comment:
@vladislav-horbatiuk in case you didn't know, if you leave `add_library` without a type (without `STATIC` nor `SHARED`), it will automatically build as shared if `BUILD_SHARED_LIBS` is set to on/true, and will automatically build as static otherwise. This change is redundant and unecessary since cmake does exactly that already.